### PR TITLE
Add release notes for 16.2.72

### PR DESCRIPTION
**Pull request by tas50**
Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes/> for the official Chef release notes.
 
+# What's New in 16.2.72
+
+- Habitat packages for Chef Infra Client 16 are now published with full support for the `powershell_exec` helper now added.
+- Added a new `clear` action to the `windows_user_privilege` resource.
+- Resolved a regression in Chef Infra Client 16.1 and later that caused failures running on FIPS enabled systems.
+- Resolved failures in the archive_file resource when running on Windows hosts.
```
Comment (phiggins) [inline RELEASE_NOTES.md:8]:
```suggestion
- Resolved failures in the `archive_file` resource when running on Windows hosts.
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+- Resolved a failure when running `chef-apply` with the `-j` option. Thanks [@komazarari](https://github.com/komazarari).
+- Chef Infra Client running within GitHub Actions is now properly identified as running in a Docker container. Thanks [@jaymzh](http://github.com/jaymzh).
+- SSH connections are now reused improving the speed of knife bootstrap and remote resource on slow network links. Thanks [@tecracer-theinen](https://github.com/tecracer-theinen).
```
Comment (phiggins) [inline RELEASE_NOTES.md:11]:
```suggestion
- SSH connections are now reused improving the speed of knife bootstrap and remote resources on slow network links. Thanks [@tecracer-theinen](https://github.com/tecracer-theinen).
```

Comment (IanMadd) [inline RELEASE_NOTES.md:11]:
```suggestion
- SSH connections are now reused, improving the speed of knife bootstrap and remote resource on slow network links. Thanks [@tecracer-theinen](https://github.com/tecracer-theinen).
```

Comment (tas50) [reply RELEASE_NOTES.md]: Fixed

**Pull request merged**